### PR TITLE
Remove the refreshableWallet and use avalanchego's wallet

### PR DESCRIPTION
**Pull request by holisticode**

### CODE REVIEW

File: local/blockchain.go [hunk 1]
```diff
@@ -137,7 +139,7 @@ func (ln *localNetwork) installCustomVMs(
 			if _, err := platformvm.Codec.Unmarshal(subnetTxBytes, &subnetTx); err != nil {
 				return nil, fmt.Errorf("couldn not unmarshal tx for subnet %q: %w", subnetID.String(), err)
 			}
-			pTXs[subnetID] = &subnetTx
+			pTXs = append(pTXs, subnetID)
```
Comment (felipemadero) [inline local/blockchain.go:142]: it seems most part of this for is not needed anymore (obtaining the tx, unmarshall, etc)

Comment (felipemadero) [reply local/blockchain.go:142]: probably update the comments and also the var name with txIDs ? subnetTxIDs ?

File: local/blockchain.go [hunk 2]
```diff
@@ -295,33 +298,39 @@ func (ln *localNetwork) setupWalletAndInstallSubnets(
...
 		println()
 		color.Outf("{{green}}reconnecting the wallet client after restart{{/}}\n")
 		clientURI, err := ln.getClientURI()
 		if err != nil {
-			return nil, err
+			return nil, nil, err
+		}
+		testKeychain := secp256k1fx.NewKeychain(genesis.EWOQKey)
+		allTxs := append(pTXs, subnetIDs...)
+		baseWallet, err = primary.NewWalletWithTxs(ctx, clientURI, testKeychain, allTxs...)
```
Comment (felipemadero) [inline local/blockchain.go:324]: is it possible to reuse setupWallet here?

File: local/blockchain.go [hunk 3]
```diff
@@ -249,7 +252,7 @@ func (ln *localNetwork) setupWalletAndInstallSubnets(
 	}
 	platformCli := platformvm.NewClient(clientURI)
 
-	pTXs := make(map[ids.ID]*txs.Tx)
+	pTXs := make([]ids.ID, 0)
```
Comment (felipemadero) [inline local/blockchain.go:255]: []ids.ID{} ?

File: local/blockchain.go (hunk 1 above)
Comment (holisticode) [reply local/blockchain.go:142]: You're very right with the first comment, thanks.
Regarding the second, I'd like to leave it as subnetID, it's a good reminder that the subnet ID is actually also the tx ID. Finally, pTXs was also used inside avalanchego, so I'd like to keep it this way.

File: local/blockchain.go (hunk 3 above)
Comment (holisticode) [reply local/blockchain.go:255]: ok

File: local/blockchain.go (hunk 2 above)
Comment (holisticode) [reply local/blockchain.go:324]: It is, but setupWallet is also doing more stuff like checking balance which seems unnecessary here

**Pull request merged**